new_platforms/nuget/build/native/openenclave.targets

```diff
@@ -0,0 +1,126 @@
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+    <!-- Relative paths to the Open Enclave SDK Libraries within the NuGet Package -->
+    <PropertyGroup>
+        <SgxHwX86Dbg>$(MSBuildThisFileDirectory)..\..\lib\native\v141\sgx\hw\x86\Debug</SgxHwX86Dbg>
+        <SgxHwX64Dbg>$(MSBuildThisFileDirectory)..\..\lib\native\v141\sgx\hw\x64\Debug</SgxHwX64Dbg>
+        <SgxSimX86Dbg>$(MSBuildThisFileDirectory)..\..\lib\native\v141\sgx\sim\x86\Debug</SgxSimX86Dbg>
+        <SgxSimX64Dbg>$(MSBuildThisFileDirectory)..\..\lib\native\v141\sgx\sim\x64\Debug</SgxSimX64Dbg>
+        <TzHwArmDbg>$(MSBuildThisFileDirectory)..\..\lib\native\v141\tz\hw\arm\Debug</TzHwArmDbg>
+        <TzSimX86Dbg>$(MSBuildThisFileDirectory)..\..\lib\native\v141\tz\sim\x86\Debug</TzSimX86Dbg>
+    </PropertyGroup>
+
+    <!-- Compilation Rules for Enclaves and Host Applications -->
+    <ItemDefinitionGroup>
+        <ClCompile>
+            <AdditionalIncludeDirectories>
+                $(MSBuildThisFileDirectory)include;%(AdditionalIncludeDirectories)
+            </AdditionalIncludeDirectories>
+        </ClCompile>
+    </ItemDefinitionGroup>
+
+    <!-- Compilation Rules for Enclaves Only -->
+    <ItemDefinitionGroup Condition="'$(OEType)' == 'Enclave'">
+        <ClCompile>
+            <!-- SGX Hardware and Simulation --> 
+            <PreprocessorDefinitions Condition="'$(Configuration)' == 'SGX-Debug' Or '$(Configuration)' == 'SGX-Simulation-Debug'">
+                OE_USE_SGX;OE_NO_SAL;%(PreprocessorDefinitions)
+            </PreprocessorDefinitions>
+
+            <!-- ARM TrustZone Simulation --> 
+            <PreprocessorDefinitions Condition="'$(Configuration)' == 'ARMTZ-Simulation-Debug'">
+                OE_SIMULATE_OPTEE;%(PreprocessorDefinitions)
+            </PreprocessorDefinitions>
+        </ClCompile>
+    </ItemDefinitionGroup>
+
+    <!-- Compilation Rules for Hosts Only -->
+    <ItemDefinitionGroup Condition="'$(OEType)' == 'Host'">
+        <ClCompile>
+            <!-- SGX Hardware and Simulation --> 
+            <PreprocessorDefinitions Condition="'$(Configuration)' == 'SGX-Debug' Or '$(Configuration)' == 'SGX-Simulation-Debug'">
+                OE_USE_SGX;%(PreprocessorDefinitions)
+            </PreprocessorDefinitions>
+
+            <!-- ARM TrustZone Hardware --> 
+            <PreprocessorDefinitions Condition="'$(Configuration)' == 'ARMTZ-Debug'">
+                OE_USE_OPTEE;%(PreprocessorDefinitions)
```

Comment:
OE_USE_OPTEE should not be needed any more either, it's set automatically by openenclave/host.h and openenclave/enclave.h

Reply:
Idem.